packages/dicomImageLoader/src/imageLoader/wadors/metaData/metaDataProvider.ts

```diff
@@ -123,7 +123,7 @@ function metaDataProvider(type, imageId) {
   }
 
   if (type === MetadataModules.NM_MULTIFRAME_GEOMETRY) {
-    const modality = getValue(metaData['00080060']);
+    const modality = getValue(metaData['00080060']) as string;
```

Comment:
Now that I see it a second time, I'd like the const exported as an enum somewhere.